### DISCUSSION TITLE
PDFViewer will not expand to whole screen

### DISCUSSION THREAD

**Discussion by kftb** [diegomura/react-pdf#1978] [General]
Hi all! 

I ran into a weird issue that is either a bug or so very simple to solve and I've been staring at the screen for too long. 

I am using NextJS and my tool creates a PDF and I'd like to display it in the PDFViewer. The Viewer loads but only takes up a small part of the screen. Whenever I change the 'width' and 'height' attribute with relative values (100%, 100vh), it won't take it. 

The only way to force it, is to put specific pixel values in it, but that defeats the purpose of being responsive to the screen size. 

Sandbox that reproduces my issue: https://stackblitz.com/edit/nextjs-su5bi1?file=pages/index.js

Does anyone have an idea why this is happening?

![Screenshot 2022-08-19 at 22 56 51](https://user-images.githubusercontent.com/11639100/185731253-59622a95-b7f4-450b-9d8f-5bec0573cfd6.png)

**Comment by raman-spyne** [comment]
set height and width using style

`<PDFViewer children={<Doc/>} style={{width: '100%', height: '100vh'}} filename='hello.pdf' />`
